common/components/filter/_filter.scss

```diff
@@ -0,0 +1,76 @@
+.app-filter {
+  margin: 0 0 govuk-spacing(3);
+}
+
+.app-filter__list {
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+
+  flex-wrap:wrap;
+  flex-direction:row;
+
+  @include govuk-font($size: 19, $weight: normal);
+  @include govuk-media-query($until: 370px) {
```

Comment:
Would it be better to use some of the predefined dimensions rather than creating our own?

Reply:
This is the dimension that makes it work. The predefined ones always ended up with a stack of 2. We may want to revisit this in the future when we have 4 boxes.